pact-lang-api.js

```diff
@@ -212,14 +212,20 @@ var enforceArray = function(val,msg) {
   }
 }
 
-
 /**
  * Make a full 'send' endpoint exec command. See 'prepareExecCmd' for parameters.
  */
 var simpleExecCommand = function(keyPairs, nonce, pactCode, envData, meta) {
   return mkPublicSend(prepareExecCmd(keyPairs, nonce, pactCode, envData, meta));
 };
 
+/**
+ * Make a full 'local' endpoint exec command. See 'prepareExecCmd' for parameters.
+ */
+var simpleLocalCommand = function(keyPairs, nonce, pactCode, envData, meta) {
+  return prepareExecCmd(keyPairs, nonce, pactCode, envData, meta);
+};
+
```

Comment:
This is a duplicate of `prepareExecCmd`. We can look at a renaming of that function if you want but we don't need a dupe.

Reply:
Removed the dupe, but exported it as Pact.simple.exec.createLocalCommand to keep consistent with create...Command functions.